src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,137 @@
+.page__content {
+  width: 944px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: auto;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__buy {
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    padding: 11px 55px;
+
+    &:hover {
+      background-color: #fff;
+      border-color: #00acdc;
+      color: #00acdc;
+    }
+  }
+}
+
+.catalog__card {
+  margin: 23px 24px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:nth-child(-n+4) {
+    margin-top: 50px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(4) {
+    margin-left: 0;
+  }
+}
+
+.rating {
+  &__reviews {
+    display: flex;
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+  }
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    background: url(../images/star.svg) no-repeat center center;
+    height: 16px;
+    width: 16px;
+  }
+
+  &__stars--active-1 .rating__star:nth-child(-n+1),
+  &__stars--active-2 .rating__star:nth-child(-n+2),
+  &__stars--active-3 .rating__star:nth-child(-n+3),
+  &__stars--active-4 .rating__star:nth-child(-n+4),
+  &__stars--active-5 .rating__star:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.price {
+  &__text {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__value {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+}
```

Comment:
All your blocks must be in different .scss files.

Reply:
It seems that in the task wasn`t that having all the blocks must be in a separate files.